client/my-sites/plugins/style.scss

```diff
@@ -247,6 +247,10 @@ $plugin-details-header-padding: 100px;
 			padding: 12px 16px;
 			background-color: var( --studio-pink-50 );
```

Comment:
Let's refactor away from the specific colors. We should use `var(--color-accent)` so that it displays the correct color depending on color scheme.